ferrilata-devilline/Startup.cs

```diff
@@ -75,6 +75,9 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
 
+            services.AddSwaggerGen(options =>
+                options.SwaggerDoc("v1", new Swashbuckle.AspNetCore.Swagger.Info { Title = "Level-Up", Version = "v1" })
```

Comment:
You can simplify it with `using` the package and create "only" with `new Info()`